packages/components/src/slot-fill/bubbles-virtually/slot-fill-provider.js

```diff
@@ -17,12 +17,12 @@ import isShallowEqual from '@wordpress/is-shallow-equal';
 import SlotFillContext from './slot-fill-context';
 
 function useSlotRegistry() {
-	const slots = proxyMap();
-	const fills = proxyMap();
+	const slots = useRef( proxyMap() );
+	const fills = useRef( proxyMap() );
```

Comment:
Was this needed to avoid passing `slots/fills` as a dependency for `useCallback`?

Reply:
it was more to keep the same "instance" of the proxy across rerenders. I stole this approach from their handbook https://github.com/pmndrs/valtio/wiki/How-to-use-with-context

Reply:
That's a nice trick 👍